django/test/runner.py

```diff
@@ -581,6 +581,18 @@ def add_arguments(cls, parser):
             ),
         )
 
+    def log(self, msg, level=None):
```

Comment:
Do we need any validation for level? What happens if I pass in say -1 or "1" into level?

Reply:
Verbosity is already validated `django.core.management.base.BaseCommand`, I don't think we need more than that.